Makefile

```diff
@@ -1,11 +1,18 @@
 # SPDX-License-Identifier: GPL-2.0
 KERNELRELEASE ?= $(shell uname -r)
+KMAJ = $(shell echo $(KVER) | \ sed -e 's/^\([0-9][0-9]*\)\.[0-9][0-9]*\.[0-9][0-9]*.*/\1/') 
```

Comment:
What is the value of KVER in a qemu-user environment?

This seems to succeed because sed is not installed.
https://github.com/pguyot/cr14/runs/5834361272?check_suite_focus=true#step:3:483

Reply:
This logic was probably copied from somewhere else and is broken here.

The way this works is as follows: if nothing is provided, KERNELRELEASE is set to `uname -r`, to get the current kernel's version. Within CI, either [https://github.com/pguyot/cr14/blob/master/.github/workflows/arm-runner.yml#L57](here) or in [pynab's setup script](https://github.com/nabaztag2018/pynab/blob/master/setup#L194), KERNELRELEASE is set to match the installed headers on the disk image (so the disk image works with various Pis).

KVER is not passed at all, and your change worked by accident because of suspicious `\ sed` code.

Nevertheless, knowing if this is 4.x or 5.x can be done GNU Make text manipulation functions, without using sed:

```
# Gracefully supporting the new always-y without cutting off older target with kernel 4.x
ifeq ($(firstword $(subst ., ,$(KERNELRELEASE))),4)
	always := $(dtbo-y)
else
	always-y := $(dtbo-y)
endif
```

Favoring 4 to use `always` variable and others (really meaning 5 and above for now) to use `always-y`.

Reply:
Good catch. The KVER should have been KERNELVERSION in this Makefile.
I ended up adjusting the commit to match your simpler version of it using firstword.

The remaining question is: should I implement the same always/always-y selection on the other 3 drivers PRs ?